src/control-manager.cpp

```diff
@@ -427,8 +443,8 @@ namespace dynamicgraph
         if(jointName=="all")
         {
           stringstream ss;
-          for(int i=0; i<N_JOINTS; i++)
-            ss<<JointUtil::get_name_from_id(i)<<" "<<m_jointCtrlModes_current[i]<<"; ";
+          for(unsigned int i=0; i<m_nJoints; i++)
+            ss<<m_robot->model().getJointName(i)<<" "<<m_jointCtrlModes_current[i]<<"; ";
```

Comment:
Fix in commit bc5f539678009aa87d3eaa2a26e524133f5d03dd. But IMHO adding a remapping between name and id in a complex robotics application, in the long run, may just generate another set of confusion for newcomers...